pkg/virtualization/core/host/hostcomputersystem.go

```diff
@@ -259,3 +259,44 @@ func (hc *HostComputerSystem) GetDefaultEthernetSwitchPortVLANSettingData(vlanId
 
 	return spps, nil
 }
+
+func (hc *HostComputerSystem) GetDefaultEthernetSwitchPortIsolationSettingData(vlanId uint32) (*switchport.EthernetSwitchPortIsolationSettingData, error) {
+	tmp, err := hc.GetDefaultPortSettingData("Ethernet Switch Port Isolation Settings", "Msvm_EthernetSwitchPortIsolationSettingData")
+	if err != nil {
+		return nil, err
+	}
+
+	spps, err := switchport.NewEthernetSwitchPortIsolationSettingData(tmp)
+	if err != nil {
+		return nil, err
+	}
+
+	defer func() {
+		if err != nil {
+			spps.Close()
+			spps = nil
+		}
+	}()
+
+	err = spps.SetPropertyDefaultIsolationId(vlanId)
+	if err != nil {
+		return nil, err
+	}
+	// VLAN 3
```

Comment:
Out of curiosity, is this always going to be VLAN 3? If so, can you make this a constant so that the people better understand what this value is when they read the code in the future?

Reply:
copy pasted GetDefaultEthernetSwitchPortVLANSettingData from above. :) agree, i will make it a constant.